wit/caching.wit

```diff
@@ -0,0 +1,75 @@
+// The `wasi:keyvalue/cache` interface provides the operations for using a
+// single instance of a cache, which is a non-durable, weakly-consistent
```

Comment:
I think a more common word for "weakly consistent" is "eventual consistency"

Reply:
For a cache, I think we need something even weaker than eventual consistency: eventual consistency suggests that, given enough time, all clients will eventually see the same thing while, in a distributed cache, even that's not true.

Reply:
I wouldn't even use the term "consistency" since it's liable to be conflated with "cache consistency".